app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskViewModel.kt

```diff
@@ -37,7 +38,8 @@ import kotlinx.coroutines.launch
  * how to deal with more complex scenarios.
  */
 class AddEditTaskViewModel(
-    private val tasksRepository: TasksRepository
+    private val getTaskUseCase: GetTaskUseCase,
```

Comment:
To make it simpler and read better, you can call these `getTask` and `save` without the `UseCase` suffix

Reply:
I think for an example this is more clear, but don't have strong feelings about it.